keyboards/ergodox/infinity/drivers/gdisp/IS31FL3731C/gdisp_IS31FL3731C.c

```diff
@@ -204,7 +204,8 @@ LLDSPEC bool_t gdisp_lld_init(GDisplay *g) {
 		uint8_t* src = PRIV(g)->frame_buffer;
 		for (int y=0;y<GDISP_SCREEN_HEIGHT;y++) {
 		    for (int x=0;x<GDISP_SCREEN_WIDTH;x++) {
-		        PRIV(g)->write_buffer[get_led_address(g, x, y)]=CIE1931_CURVE[*src];
+		        uint8_t val = (uint16_t)*src * g->g.Backlight / 100;
```

Comment:
@fredizzimo using *src directly didn't work. However, I was able to hardcode a value (I used 240) and the LEDs turned on correctly. I'm not really sure what this code is trying to do, as it doesn't seem correct to me; what is `src`? afaict, we should be trying to scale val to be between 0-255, as that's the size iof the CIE1931_CURVE array, right? Or am I misunderstanding something here?